benchmarks/AiDotNet.PyTorchParity/pytorch/benchmark.py

```
@@ -0,0 +1,345 @@
+        for _ in range(iterations):
+            start = time.perf_counter()
+            model(x)
+            synchronize(device)
+            elapsed = time.perf_counter() - start
+            steady.append(elapsed)
+            rss_peak = max(rss_peak, process.memory_info().rss / 1024 / 1024)
+        total = sum(steady)
+        cuda_peak = torch.cuda.max_memory_allocated(device) / 1024 / 1024 if device.type == "cuda" else 0.0
+        # p95 latency: robust to rig-contention noise that swings the mean.
+        steady_sorted = sorted(steady)
+        p95_idx = min(len(steady_sorted) - 1, int(round(0.95 * (len(steady_sorted) - 1))))
```

Ruff (0.15.14)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)